src/main/java/com/o19s/es/ltr/feature/store/FeatureNormDefinition.java

```diff
@@ -0,0 +1,41 @@
+package com.o19s.es.ltr.feature.store;
+
+import com.o19s.es.ltr.ranker.normalizer.Normalizer;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.ToXContent;
+
+import java.io.IOException;
+
+/**
+ * Parsed feature norm from model definition
+ */
+public interface FeatureNormDefinition extends ToXContent {
+
+    /**
+     * @return
+     *  Construct the feature norm associated with this definitino
+     */
+    Normalizer createFeatureNorm();
+
+    /**
+     * @return
+     *  The feature name associated with this normalizer to
+     *  later associate with an ord
+     */
+    String featureName();
+
+    /**
+     * @return
+     *  A type of normalizer
+     */
+    StoredFeatureNormalizers.Type normType();
+
+    /**
+     * Serialize to a StreamOutput
+     * @param out
+     *  where the ftr norm defn will be serialized
+     * @throws
+     *  IOException if issues occur with writing to output
+     */
+    void writeTo(StreamOutput out) throws IOException;
```

Comment:
you should probably extend org.elasticsearch.common.io.stream.Writeable as well so that you don't have to re-declare this method.